.github/workflows/reusable-build.yaml

```
@@ -0,0 +1,95 @@
+name: "Build Programs"
+
+on:
+  workflow_call:
+    inputs:
+      commit-hash:
+        description: "The commit hash to build from"
+        type: string
+        required: true
+      cf-config-bucket-root-key:
```

```suggestion
      environment:
```
give it a meaningful name wrt to this action, even if the value gets used as part of some other variable later?